plugins/org.jkiss.dbeaver.ext.mssql/src/org/jkiss/dbeaver/ext/mssql/model/SQLServerDialect.java

```diff
@@ -42,10 +43,10 @@
 
 import java.util.*;
 
-public class SQLServerDialect extends JDBCSQLDialect implements TPRuleProvider {
+public class SQLServerDialect extends JDBCSQLDialect implements TPRuleProvider, SQLDialectDDLExtension {
 
     private static final String[][] TSQL_BEGIN_END_BLOCK = new String[][]{
-        {SQLConstants.BLOCK_BEGIN, SQLConstants.BLOCK_END}
+            {SQLConstants.BLOCK_BEGIN, SQLConstants.BLOCK_END}
```

Comment:
Please fix your formatting configuration. The `continuation indent` should be `4`, you have `8`.

Reply:
fixed